docs/pages/router/static-rendering/index.mdx

```diff
@@ -0,0 +1,222 @@
+}
+
+export default function Page() {
+  const { id } = useLocalSearchParams();
+
+  return <Text>Post {id}</Text>;
+}
+```
+
+This will output a file for each post in the `dist` folder. For example, if the `generateStaticParams` method returned `[{ id: "alpha" }, { id: "beta" }]`, the following files would be generated:
+
+<FileTree files={['dist/blog/alpha.html', 'dist/blog/beta.html']} />
+
+## Root HTML
+
+By default, every page is wrapped with some small HTML boilerplate, this is known as the **root HTML**.
+
+You can customize the root HTML file by creating a **app/+html.tsx** file in your project. This file exports a React component that only ever runs in Node.js, meaning global CSS cannot be imported inside of it. The component will wrap all routes in the **app** directory. This is useful for adding global `<head>` elements or disabling body scrolling.
+
+> Notice: Global context providers should go in the [Root Layout](/router/advanced/root-layout) component, not the Root HTML component.
+
+```js app/+html.tsx
+import { ScrollViewStyleReset } from 'expo-router/html';
+
+// This file is web-only and used to configure the root HTML for every
+// web page during static rendering.
+// The contents of this function only run in Node.js environments and
+// do not have access to the DOM or browser APIs.
+export default function Root({ children }: { children: React.ReactNode }) {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+
+        {/* 
+          This viewport disables scaling which makes the mobile website act more like a native app.
+          However this does reduce built-in accessibility. If you want to enable scaling, use this instead:
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
+        */}
+        <meta
+          name="viewport"
+          content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1.00001,viewport-fit=cover"
+        />
+        {/* 
+          Disable body scrolling on web. This makes ScrollView components work closer to how they do on native. 
+          However, body scrolling is often nice to have for mobile web. If you want to enable it, remove this line.
+        */}
+        <ScrollViewStyleReset />
+
+        {/* Add any additional <head> elements that you want globally available on web... */}
+      </head>
+      <body>{children}</body>
+    </html>
+  );
+}
+```
+
+- The `children` prop comes with the root `<div id="root" />` tag included inside.
+- The JavaScript scripts are appended after the static render.
+- React Native web styles are statically injected automatically.
+- Global CSS should not be imported in this file, instead use the [Root Layout](/router/advanced/root-layout) component.
+- Browser APIs like `window.location` are not available in this component as it only runs in Node.js during static rendering.
```

Comment:
It seems to me that these bullet points are like summary since we are repeating points like: "Global CSS should not be imported in this file, instead use the [Root Layout](/router/advanced/root-layout) component.". Can we add a new subsection or line that this is a summarization (to provide people context of what they are all about).

```suggestion
- The `children` prop comes with the root `<div id="root" />` tag included inside.
- The JavaScript scripts are appended after the static render.
- React Native web styles are statically injected automatically.
- Global CSS should not be imported into this file. Instead, use the [Root Layout](/router/advanced/root-layout) component.
- Browser APIs like `window.location` are unavailable in this component as it only runs in Node.js during static rendering.
```